contents/handbook/growth/sales/yc-onboarding.md

```diff
@@ -4,39 +4,53 @@ sidebar: Handbook
 showTitle: true
 ---
 
-As a Y Combinator company ourselves, we want to support other YC companies that wish to use PostHog. As such, our onboarding process for YC companies is different and more extensive than our general process.
+We want to support other YC companies using PostHog because:
+
+- Many of these companies will quickly grow into [our ICP](https://posthog.com/handbook/strategy/ideal-customer-persona) and we have an opportunity to get in early with them. 
+- A lot of our most helpful and direct product feedback has generally come from other YC companies. 
+- It's nice to give back to the YC community. 
 
 ## The YC deal
 
-Our YC deal provides the following:
+For startups in the W23 batch:
+
+- Get PostHog Cloud free for 12 months with up to 20m events and 50,000 session recordings per month
+- Choose between a free pair of Airpods or a Timbuk2 backpack
+- Book into a free YC group setup session where we will…
+  - Build a dashboard for the batch with your product growth metrics
+  - Pair with you to ensure you pass calls correctly
+- Get the CEO’s phone number and message him any time - seriously (well maybe not any time, he does have to sleep you know…)
+
+For previous batches:
+
+- Get PostHog Cloud free for 6 months with up to 20m events and 50,000 session recordings per month
 
-- 20 million events/month, free for 6 months (PostHog Cloud or Self Hosted)
-- Companies also get a $50 gift card to spend on our merch and dedicated support (i.e. a private Slack with our core team members).
+> Only James has the ability to update our deal in Bookface, so any changes to the deal text should be made in [this Google Doc](https://docs.google.com/document/d/17MtngAx2DeVo3YyhPwwKRvakI54SJuNmHkPtFns2IVQ/edit) then shared with him. 
 
-For current batch YC companies, we also offer:
+## Our process
 
-- 20 million events/month, free for a year (instead of 6 months)
-- Free group pairing session to help with setup
-- PostHog AirPods, a PostHog Timbuk2 Backpack, a $150 Open Collective gift card, or a $150 PostHog Merch gift card.
-- A referral scheme that lets you double-down on the above deal
+We want to make this feel as personalized as possible - there is an outsized impact in giving YC companies personal attention to ensure they are successful with PostHog. 
 
-> **Note:** Batch-specific deals will be updated here as new batches come along.
+- Kendal maintains a sheet with the details for all companies in the current batch
```

Comment:
```suggestion
- Kendal maintains a sheet with the details for all companies in the current batch as they launch (since the list isn't available anywhere)
```